src/Demo_Attendee_List/Shortcodes/Attendee_List.php

```diff
@@ -172,16 +178,22 @@ protected function enqueue_assets(): void {
 	 * @return string
 	 */
 	public function output(): string {
-		$this->enqueue_assets();
+		/*
+		 * Action that the assets are enqueued on. Allows developers to unload assets.
+		 *
+		 * @since 1.0.0
+		 */
+		do_action( 'sdokus_demo_attendee_list_shortcode_before_output', $this );
+
+		// @todo - Move this to a template.
 		ob_start();
 		?>
         <div class="test" data-id="<?php echo esc_attr( $this->attributes['id'] ); ?>">
-            <p>
-                TESTING TESTING 123
-            </p>
+            <h2>
+	            <?php __('All Attendees', 'sdokus-demo-attendee-list'); ?>
```

Comment:
This wont echo, make sure to test the functionality after applying changes. Lets add escaping here with `esc_html_e()`